docs/contributors/WordPress Release Integration

```diff
@@ -0,0 +1,38 @@
+With each WordPress release cycle, various versions of Gutenberg are included. This has created confusion over time as people try to figure out how best to debug problems and report bugs appropriately. To make this easier for everyone to keep track of, this document serves as a canonical list of the Gutenberg versions present in each WordPress release.  
+
```

Comment:
It may also be good to note that each WordPress release also includes bug fixes from later Gutenberg releases that occur during the beta period. It's a little tricky to describe in non-technical terms, maybe something like:

"During the beta period, additional bug fixes from later Gutenberg releases than those noted are cherry-picked into the WordPress release"

Reply:
Added that phrasing in. As for an overview of the beta process, I'm conflicting as that should probably belong on other pages 🤔